docs/data/sdks/typescript-react-native/migration.md

```diff
@@ -3,6 +3,329 @@ title: React Native SDK Migration Guide
+    
+    const groupType = 'plan';
+    const groupName = 'enterprise';
+    const identifyObj = new Identify()
+    identifyObj.set('key1', 'value1');
+
+    Amplitude.getInstance().groupIdentify(groupType, groupName, identifyObj);
+    ```
+
+=== "@amplitude/analytics-react-native"
+
+    ```typescript
+    import { Identify, groupIdentify } from '@amplitude/analytics-react-native';
+
+    const groupType = 'plan';
+    const groupName = 'enterprise';
+    const identifyObj = new Identify()
+    identifyObj.set('key1', 'value1');
+
+    groupIdentify(groupType, groupName, identifyObj);
+    ```
+
+### Tracking revenue
+
+#### `logRevenue()`
+
+The `logRevenue()` API maps to `revenue()`. `receipt` and `receiptSignature` is not supported.
+
+=== "@amplitude/react-native"
+
+    ```typescript
+    import { Amplitude } from '@amplitude/react-native';
+    
+    const userProperties = {
+        price: 3,
+        productId: 'com.company.productId',
+        quantity: 2,
+        revenueType: 'productRevenue',
+        eventProperties: {
+            key: 'value',
+        },
+    };
+
+    ampInstance.logRevenue(userProperties);
+    ```
+
+=== "@amplitude/analytics-react-native"
+
+    ```typescript
+    import { Revenue, revenue } from '@amplitude/analytics-react-native';
+
+    const event = new Revenue()
+        .setPrice(3)
+        .setProductId('com.company.productId')
+        .setQuantity(2)
+        .setRevenueType('productRevenue')
+        .setEventProperties({
+            key: 'value',
+        })
+
+    revenue(event);
+    ```
+You can also `setRevenue(6)` instead of `setPrice(3)` and `setQuantity(2)`.
+
+## Advanced topics
+
+### Identity
+
+#### Device ID
+
+As the maintenance React Native SDK is a wrapper of the maintenance iOS, maintenance Android SDK and maintenance Browser SDK and provides mappings from React Native to native SDK functions, device ID generation follows the native SDK of each platform. Learn more about device ID lifecycle of [maintenance iOS SDK](../../ios/#device-id-lifecycle) and [maintenance Android SDK](../../android/#device-id-lifecycle). You can also call `setAdvertisingIdForDeviceId()` or `setAppSetIdForDeviceId()` to set ADID or App Set ID as device ID.
+
+The new React Native SDK initializes the device ID in the following order, with the device ID being set to the first valid value encountered:
+
+1. Device ID of in the configuration on initialization
+2. "deviceId" value from URL param, for example http://example.com/?deviceId=123456789. If it runs on Web.
+3. Device ID in cookie storage.
+4. A randomly generated 36-character UUID
+
+#### Advertising IDs
+
+Maintenance React Native SDK supports setting an advertising ID as device ID by `setAdvertisingIdForDeviceId()` or `setAppSetIdForDeviceId()`. The new React Native SDK tracks ADID  by default as `config.trackingOptions.adid` defaults to `true`. However, the new React Native SDK doesn't support App Set ID, IDFA, or IDFV.
```

Comment:
> App Set ID, IDFA, or IDFV

As mentioned this is WIP. Should be in prod early next week

Reply:
We will not directly support retrieving IDFA, but customers will be still able to configure the trackingoption, and set the `event.idfa` themselves if they want that functionality.

Reply:
Add an admonition like in COPPA section. Will change it once https://github.com/amplitude/Amplitude-TypeScript/pull/456 releases